src/status_im2/contexts/browser/home/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im2.contexts.browser.home.view
+  (:require
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [utils.i18n :as i18n]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [status-im2.common.home.top-nav.view :as common.top-nav]
+    [status-im2.common.home.title-column.view :as common.title-column]
+    [status-im2.contexts.browser.home.style :as style]
+    [utils.re-frame :as rf]))
+
+(def dapp-favorites
+  [{:logo (quo.resources/get-dapp :coingecko) :name "CoinGecko"}
+   {:logo (quo.resources/get-dapp :aave) :name "Aave"}
+   {:logo (quo.resources/get-dapp :1inch) :name "1inch"}
+   {:logo (quo.resources/get-dapp :zapper) :name "Zapper"}
+   {:logo (quo.resources/get-dapp :uniswap) :name "Uniswap"}
+   {:logo (quo.resources/get-dapp :zerion) :name "Zerion"}])
```

Comment:
This hardcoded array of dapps is repeated here and in quo_preview because once the functionality is developed this should be removed, but the other one should remain. Does it make sense?

Reply:
that's fine, it's better to add the mock data in the preview screens and not in the quo2 components 👍